ParkingLotApi/Filters/IdNotExistExceptionFilter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using ParkingLotApi.Exceptions;
+
+namespace ParkingLotApi.Filters
+{
+    public class IdNotExistExceptionFilter : IActionFilter, IOrderedFilter
+    {
+        public int Order => int.MaxValue - 10;
```

Comment:
Actually, IOrderedFilter define exception throw order, you can adjust it according situation.